openapi-cli/src/test/resources/ballerina-to-openapi/expected_gen/data_type/map.yaml

```diff
@@ -0,0 +1,81 @@
+openapi: 3.0.1
+info:
+  title: PayloadV
+  version: 0.0.0
+servers:
+  - url: "{server}:{port}/payloadV"
+    variables:
+      server:
+        default: http://localhost
+      port:
+        default: "9090"
+paths:
+  /locations:
+    get:
+      operationId: getLocations
+      responses:
+        "200":
+          description: Ok
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Location'
+components:
+  schemas:
+    Link:
+      required:
+        - href
+      type: object
+      properties:
+        rel:
+          type: string
+        href:
+          type: string
+        types:
+          type: array
+          items:
+            type: string
+        methods:
+          type: array
+          items:
+            type: string
+    Location:
+      required:
+        - _links
+        - addressCode
+        - id
+        - name
+      type: object
+      properties:
+        _links:
+          type: object
+          additionalProperties:
+            $ref: '#/components/schemas/Link'
+        name:
+          type: object
+          additionalProperties:
+            type: string
+        id:
+          type: object
+          additionalProperties:
+            type: integer
+            format: int64
+        addressCode:
+          type: object
+          additionalProperties:
+            type: number
+            format: float
+        item:
+          type: object
+          additionalProperties: true
+        mapArray:
```

Comment:
since this is actually an array map, shall we also include a map array (`map<string>[]`) to the test (in addition to this)?

Reply:
the intention was to handle map with array type  `map<string[]>` will add array item type `map<string>`

Reply:
Add a test [a940070](https://github.com/ballerina-platform/openapi-tools/pull/1035/commits/a940070c981675395e8b3dc1fc2c407b49f79777)